build.gradle

```
@@ -13,6 +14,9 @@ repositories {
 }
 
 dependencies {
+	asciidoctor 'org.springframework.restdocs:spring-restdocs-asciidoctor:2.0.4.RELEASE'
+	testImplementation 'org.springframework.restdocs:spring-restdocs-mockmvc:2.0.4.RELEASE'
```

```suggestion
	asciidoctor 'org.springframework.restdocs:spring-restdocs-asciidoctor'
	testImplementation 'org.springframework.restdocs:spring-restdocs-mockmvc'
```

plugins에서 'io.spring.dependency-management'가 어떠한 역할을 하는지 공부해보세요~

---

'io.spring.dependency-management'가 사용하고 있는 Spring boot 버전에 맞춰 검증된 버전으로 의존성을 가져오네요! 감사합니다!